apps/api/src/utils/graphql.logger.ts

```diff
@@ -0,0 +1,61 @@
+import {
+    ApolloServerPlugin,
+    BaseContext,
+    GraphQLRequestContext,
+    GraphQLRequestContextWillSendResponse,
+    GraphQLRequestListener,
+} from "@apollo/server";
+import { Plugin } from "@nestjs/apollo";
+import { Logger } from "@nestjs/common";
+import { performance } from "perf_hooks";
+
+@Plugin()
+export class ApolloLogger implements ApolloServerPlugin {
+    private readonly logger = new Logger("GraphQL");
+
+    async requestDidStart(requestContext: GraphQLRequestContext<BaseContext>) {
```

Comment:
Mark full access specifier as the `logger`
```suggestion
    public async requestDidStart(requestContext: GraphQLRequestContext<BaseContext>) {
```